CommonInstall/util_init.pl

```diff
@@ -239,6 +239,9 @@ ($)
 	} elsif ($vendor eq "fedora") {
 		$rval=`cat /etc/fedora-release | cut -d' ' -f4`;
 		chop($rval);
+	} elsif ($vendor eq "opencloudos") {
```

Comment:
it should not reach this as we will enter block above at line 206.
So this is not needed.

Reply:
fixed.